dev/drivers/scripts/narre/plots/jevs_narre_past90days_plots.sh

```diff
@@ -42,11 +43,11 @@ export run_mpi=yes
 
 export COMIN=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver
 #export COMOUT=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver
```

Comment:
@BinbinZhou-NOAA Please remove this commented out line:
#export COMOUT=/lfs/h2/emc/vpppg/noscrub/${USER}/$NET/$evs_ver